README.md

```diff
@@ -41,17 +41,17 @@ Simply update the files under `components/(team-name)`, and open a PR with the c
 The prerequisites are:
 
 - You must have `kubectl`, `oc`, `jq`, `openssl`, `realpath` and [`yq`](https://github.com/mikefarah/yq) installed.
-- You must have `kubectl` and `oc` pointing to an existing OpenShift cluster, that you wish to deploy to. Alternatively, you can configure a local CodeReady Containers VM to deploy to.
+- You must have `kubectl` and `oc` pointing to an existing OpenShift cluster, that you wish to deploy to. Alternatively, you can configure a OpenShift Local to deploy to.
 - You must have another `kubeconfig` pointing to an existing kcp instance, that you wish to deploy to. You can use either a CPS or a local kcp instance.
 - The script `./hack/setup/install-pre-req.sh` will install these prerequisites for you, if they're not already installed.
 
-### Optional: CodeReady Containers Setup
+### Optional: OpenShift Local Setup
```

Comment:
let's put it also into at least one heading
```suggestion
### Optional: OpenShift Local (formerly CRC) Setup
```

Reply:
Done !

I've added it to the corresponding headings and in the first appearance of OpenShift Local (under `Required prerequisites` point).